Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,108 +2,55 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: var(--remotion-themed-border-width) solid var(--color-primary);
+  border-radius: var(--remotion-themed-border-radius);
+  padding: 0.4em 1.2em;
+  box-sizing: border-box;
   cursor: pointer;
-  text-decoration: none !important;
-  border: none;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  padding: 0px 2px;
-  -moz-padding-start: 0;
-  -moz-padding-end: 0;
-  outline: none;
-}
-
-button.webButton::-moz-focus-inner
-{ 
-  border: 0;
 }
 
-button.webButton span.buttonBody
+button.webButton:hover,
+button.webButton:focus
 {
-  border: solid 1px #AAAAAA;
-  font-weight: bold;
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
-  
-  display: block;
-  position: relative;
-  top: 0px;
-  left: -2px;
-  
-  width: 100%;
-  white-space: nowrap;
-
-  padding-top: 2px;
-  padding-bottom: 2px;
-  padding-left: 1px;
-  padding-right: 1px;
+  background-color: var(--color-highlight);
 }
 
-button.webButton span.buttonBody:first-child::before
+button.webButton[disabled]
 {
-  content: "";
-  height: 1.25em;
-  width: 0;
-  display: inline-block;
-  vertical-align: text-bottom;
-}
-
-button.webButton span.buttonBody img
-{
-  vertical-align: bottom;
-  border: none;
+  background-color: transparent;
```

Comment:
maybe we should make a `--color-standard-background`?

Reply:
It's already defined: `--color-standard-background: #FFFFFF;`

Reply:
okay, update: we should use white (okay) but we should use --color-standard-background here and in line 5 (button.webbutton). Thus, only the secondary button has a transparent background.

Reasoning: we use white as background for all frameed elements, so the standard button should also get a white background with it's border. Works better with the properyt consistency as well.

 And yes, maybe the name "secondary" isn't great and should be changed in rm-7928. Would it be better to move those eitems with primary and secondary button from the current changeset to a code snippet in the rm-7928 task?

Reply:
I just saw this on an input-value screenshop: the browser natively makes the background light gray instead of white (or we do it, don't know). We should consider using a light gray for the --color-disabled-background. Of cource, this could be decided later on, so long as we already have a css property for this.